File: src/main/java/com/melonbase/microquark/repo/data/Wahlresultat.java
package com.melonbase.microquark.repo.data;

import one.microstream.reference.Lazy;

import java.util.List;
import java.util.Map;

public class Wahlresultat {

  private Lazy<Map<Kanton, List<Boolean>>> stimmenByKanton;

  public Wahlresultat(Map<Kanton, List<Boolean>> stimmenByKanton) {
    this.stimmenByKanton = Lazy.Reference(stimmenByKanton);
  }

  public Map<Kanton, List<Boolean>> getStimmenByKanton() {
    return Lazy.get(stimmenByKanton);
  }
}

File: src/main/java/com/melonbase/microquark/repo/data/Wahlresultat.kt
package com.melonbase.microquark.repo.data

import one.microstream.reference.Lazy

class Wahlresultat(stimmenByKanton: Map<Kanton, List<Boolean>>) {

  private val stimmenByKanton: Lazy<Map<Kanton, List<Boolean>>> = Lazy.Reference(stimmenByKanton)

  fun getStimmenByKanton(): Map<Kanton, List<Boolean>> {
    return Lazy.get(stimmenByKanton)
  }
}